packages/core/saved-objects/core-saved-objects-server/src/type_registry.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { SavedObjectsType } from './saved_objects_type';
+
+/**
+ * Registry holding information about all the registered {@link SavedObjectsType | saved object types}.
+ */
+export interface ISavedObjectTypeRegistry {
```

Comment:
Same, the ISOTR interface was inferred from the concrete type, changed it to be an explicit one.